contracts/modules/staking/StakingProducts.sol

```diff
@@ -0,0 +1,513 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.9;
+
+import "../../abstract/MasterAwareV2.sol";
+import "../../abstract/Multicall.sol";
+import "../../interfaces/IStakingProducts.sol";
+import "../../libraries/Math.sol";
+import "../../libraries/SafeUintCast.sol";
+import "../../libraries/StakingPoolLibrary.sol";
+
+contract StakingProducts is IStakingProducts, MasterAwareV2, Multicall {
+  using SafeUintCast for uint;
+
+  uint public constant SURGE_PRICE_RATIO = 2 ether;
+  uint public constant SURGE_THRESHOLD_RATIO = 90_00; // 90.00%
+  uint public constant SURGE_THRESHOLD_DENOMINATOR = 100_00; // 100.00%
+  // base price bump
+  // +0.2% for each 1% of capacity used, ie +20% for 100%
+  uint public constant PRICE_BUMP_RATIO = 20_00; // 20%
+  // bumped price smoothing
+  // 0.5% per day
+  uint public constant PRICE_CHANGE_PER_DAY = 50; // 0.5%
+  uint public constant INITIAL_PRICE_DENOMINATOR = 100_00;
+  uint public constant TARGET_PRICE_DENOMINATOR = 100_00;
+  uint public constant MAX_TOTAL_WEIGHT = 20_00; // 20x
+
+  // The 3 constants below are also used in the StakingPool contract
+  uint public constant TRANCHE_DURATION = 91 days;
+  uint public constant MAX_ACTIVE_TRANCHES = 8; // 7 whole quarters + 1 partial quarter
+  uint public constant WEIGHT_DENOMINATOR = 100;
+
+  // pool id => product id => Product
+  mapping(uint => mapping(uint => StakedProduct)) private _products;
+  // pool id => { effectiveWeight, targetWeight }
+  mapping(uint => Weights) public weights;
```

Comment:
Confused by this.

// pool id => { effectiveWeight, targetWeight }


Assuming this is the equibalent of 

```  uint32 internal totalEffectiveWeight;
  uint32 internal totalTargetWeight;```
  
  shouldn't it use the same naming?